src/App.jsx

```diff
@@ -14,33 +15,90 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SET_SF_ALPHABETICALLY = 'alph';
```

Comment:
SET sounds like a call to action
```suggestion
const SORT_ALPHABETICALLY = 'alph';
```